sdk/storage/storage-blob/test/encrytion.spec.ts

```diff
@@ -0,0 +1,154 @@
+import * as assert from "assert";
+import * as dotenv from "dotenv";
+import {
+  getBSU,
+  recorderEnvSetup,
+} from "./utils";
+import { record } from "@azure/test-utils-recorder";
+import {
+  BlobServiceClient,
+  BlobClient,
+  BlockBlobClient,
+  ContainerClient
+} from "../src";
+import { Test_CPK_INFO } from "./utils/constants";
+import { isNode } from "@azure/core-http";
+dotenv.config({ path: "../.env" });
+
+describe("Encryption Scope", function () {
+  let blobServiceClient: BlobServiceClient;
+  let containerName: string;
+  let containerClient: ContainerClient;
+  let blobName: string;
+  let blobClient: BlobClient;
+  let blockBlobClient: BlockBlobClient;
+  const content = "Hello World";
+  const accountEncryptionKey = "$account-encryption-key";
+
+  const encryptionScopeEnvVar1 = "ENCRYPTION_SCOPE_1";
+  const encryptionScopeEnvVar2 = "ENCRYPTION_SCOPE_2";
+  let encryptionScopeName1: string | undefined;
+  let encryptionScopeName2: string | undefined;
+  if (isNode) {
+    encryptionScopeName1 = process.env[encryptionScopeEnvVar1];
+    encryptionScopeName2 = process.env[encryptionScopeEnvVar2];
+  } else {
+    encryptionScopeName1 = (window as any).__env__[encryptionScopeEnvVar1];
+    encryptionScopeName2 = (window as any).__env__[encryptionScopeEnvVar2];
+  }
+
+  let recorder: any;
+
+  before(function () {
+    if (!encryptionScopeName1 || !encryptionScopeName2) {
+      this.skip();
+    }
+  });
+
+  beforeEach(async function () {
+    recorder = record(this, recorderEnvSetup);
+    blobServiceClient = getBSU();
+    containerName = recorder.getUniqueName("container");
+    containerClient = blobServiceClient.getContainerClient(containerName);
+    blobName = recorder.getUniqueName("blob");
+    blobClient = containerClient.getBlobClient(blobName);
+    blockBlobClient = blobClient.getBlockBlobClient();
+  });
+
+  afterEach(async function () {
+    await containerClient.delete();
+    recorder.stop();
+  });
+
+  it("create container", async () => {
+    await containerClient.create();
+    let containerChecked = false;
+    for await (const container of blobServiceClient.listContainers({
+      includeMetadata: true,
+    })) {
+      if (container.name === containerName) {
+        assert.strictEqual(container.properties.defaultEncryptionScope, accountEncryptionKey);
```

Comment:
Is "$account-encryption-key" the default value set by storage service?

Reply:
Yes.